components/gravityforms/class-gf-convertkit.php

```diff
@@ -288,7 +294,7 @@ public function process_feed( $feed, $entry, $form ) {
 			}
 		}
 
-		ckgf_convertkit_api_add_email( $form_id, $entry[ $field_map_e ], $entry[ $field_map_n ], null, $fields );
+		ckgf_convertkit_api_add_email( $form_id, $email, $name, null, $fields );
```

Comment:
Are these safe to be null?

Reply:
Yep, there's a [check here](https://github.com/ConvertKit/convertkit-gravity-forms/blob/0f2ef8f03b4238e793c9fab6f06523709cc9e6bd/functions/convertkit.php#L30) in `ckgf_convertkit_api_request()` to fetch the API key if it's not set. I don't love how that's all done and will probably refactor at some point, but it's safe.